### ISSUE TITLE
[Bug]

### ISSUE THREAD

**Issue by TheAlpH3x** [MeteorDevelopment/meteor-client#848]
**Describe the bug**
New CA is tanking FPS so bad, from my usual 100+ fps now only got below 20 fps when CA is on
**To Reproduce**
1. Go to 'Click GUI'
2. Click on 'Crystal Aura'

**Expected behavior**
-
**Screenshots**
-
**Additional context**
Add any other context (meteor version, minecraft version, other mods you are using, etc).
Meteor 0.4.3 Dev build 905, Minecraft version 1.16.5, Mods Sodium,Lithium,Phosphor

**Comment by IIpho3nix** [comment]
Hello I Would Lıke The Correct This It Only Happens When Support Is On can You Try Turning Off Support

**Comment by TheAlpH3x** [comment]
yes, I have tried it with support on and off and it seems like that was the problems, thank you for the clarification

**Comment by TheAlpH3x** [comment]
> Hello I Would Lıke The Correct This It Only Happens When Support Is On can You Try Turning Off Support and Reproduce it

yes, I have tried it with support on and off and it seems like that was the problems, thank you for the clarification

**Comment by RoastedCheese** [comment]
download more ram
